lib/info.gi

```diff
@@ -32,34 +32,36 @@
 ##
 #R  IsInfoClassListRep(<obj>)                       length one positional rep
 ##                              
-##  An InfoClass is represented as a length one positional object with
-##  a positive integer in ic![1]. No other representations are 
-##  anticipated
+##  An InfoClass is represented as a positional object with the following
+##  members:
 ##
-
-DeclareRepresentation("IsInfoClassListRep", IsPositionalObjectRep,[]);
-
-#############################################################################
-##
-#V  InfoData                 record of private stuff
-#V  InfoData.CurrentLevels   the current desired verbosity levels set by user
-#V  InfoData.ClassNames      names of all info classes, used for printing
-#V  InfoData.Handler         optional handler for InfoDoPrint
+##  1 : Current level (A positive integer)
+##  2 : ClassName     (String)
+##  3 : Handler       (Optional, handler for InfoDoPrint)
+##  4 : Output        (Optional, output stream)
+##  5 : ClassNum      (An integer identifying the class)
 ##
-##  these are all lists, and the first two should have the same length, 
-##  which defines the number of Info classes that exist
+##  ClassNum is placed at the end so the length of the list never changes
```

Comment:
I'll explain it better. I don't want to put optional members at the end, because then the list of the length can change when those optional members become assigned, which doesn't work for IsAtomicPositionalObjects, which use fixed length atomic lists (that comment really is too short).